chart/values_nightly.yaml

```diff
@@ -0,0 +1,5 @@
+ingress:
+  web:
+    annotations:
+      alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:ap-northeast-1:242864343139:certificate/d6930331-0470-4d85-90c4-7b095914bf1f
```

Comment:
Created with Terraform. We need to discuss how to store & manage this kind of value further.